ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -533,14 +627,22 @@ public DevOptionHandler(string name, Action onSelect)
 
             public string Name { get; }
 
+#if WINDOWS_UWP
             public IAsyncInfo AsyncInfo { get; set; }
+#else
+            public Action AsyncInfo { get; set; }
```

Comment:
>AsyncInfo [](start = 26, length = 9)

No need to keep the name the same if you make the change below (I don't see any other shared code that references the AsyncInfo property)